sdks/java/io/elasticsearch-tests/elasticsearch-tests-5/src/test/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIOIT.java

```diff
@@ -93,24 +86,7 @@ public static void afterClass() throws Exception {
 
   @Test
   public void testSplitsVolume() throws Exception {
```

Comment:
There is no more point having a separate testSplitVolume.  Only one testSplit method is enough. The only difference between the 2 methods is desiredBundleSizeBytes = 10000. You can add this line to an else in `if (!useAsITests)` in testSplit.

Reply:
Thank you @echauchot  I was so busy last week. Today I will have time to fix all.

Reply:
Hi @echauchot , @timrobertson100 , I ran integration test for new PR #6326  and added the notes at https://gist.github.com/dattran-vn01/f14e595ed765e0bdffdc2895271864ee

Reply:
Hi @dattran-vn01 to solve your rebase issue:
1. checkout master branch to local and update it (pull --rebase) to the latest commits
2. create a new local branch from master : checkout -b BEAM-5107-ES6
3. cherry pick the commits you did on branch BEAM-5107-Support-ES-6.x-for-ElasticsearchIO to the new BEAM-5107-ES6 branch. You can cherry pick using Intellij: select the commits in git log history window of branch BEAM-5107-Support-ES-6.x-for-ElasticsearchIO and right-click cherry pick option. It will commit the cherry picked commits to the current branch  (BEAM-5107-ES6). Don't forget to remove the "cherry picked from ..." comment in git comment.
4. force push your branch BEAM-5107-ES6 to the remote branch BEAM-5107-Support-ES-6.x-for-ElasticsearchIO to overwrite it.
That way we keep review history and we can close your other PR

Reply:
Thank you @echauchot . Let me try it.

Reply:
Also when you cherry pick don't forget to solve the conflicts that might be there.

Reply:
Thank you @echauchot . I followed what you guided to solve branch problem. I will run Integration tests again for this PR.